README.md

```diff
@@ -17,8 +17,145 @@ value from data whilst ensuring it is properly governed.
 
 # Postgres Connector
 
-This project contains the Postgres connector for Egeria.
+This project currently contains the Postgres Database Integration Daemon Connector for Egeria.
```

Comment:
We have another PR for the readme where we are reviewing the text - did you see my most up to date comments?
Probably easiest to concluse that review first then rebase/merge to get this fix in?